### PR TITLE
Edge::Future updates

### PR DESCRIPTION
**Pull request by pitr-ch**
replacing #346

### CODE REVIEW

**Comment by iNecas** [comment]
I've oepend a PR to this branch to add tests for the thing https://github.com/pitr-ch/concurrent-ruby/pull/1 taken from #346

**Comment by pitr-ch** [comment]
@iNecas Thanks, I've copied them locally but forgot to commit and push it :(

**Comment by pitr-ch** [comment]
@iNecas could you pull this and check that it works for you?

**Comment by pitr-ch** [comment]
fixes #351

**Comment by iNecas** [comment]
Works for me

File: concurrent-ruby-edge.gemspec [hunk 1]
```diff
@@ -10,7 +10,7 @@ Gem::Specification.new do |s|
   s.name             = 'concurrent-ruby-edge'
   s.version          = Concurrent::EDGE_VERSION
   s.platform         = Gem::Platform::RUBY
-  s.authors          = ["Jerry D'Antonio", 'The Ruby Concurrency Team']
+  s.authors          = ["Jerry D'Antonio", 'Petr Chalupa', 'The Ruby Concurrency Team']
```
Comment (jdantonio) [inline concurrent-ruby-edge.gemspec]: :+1:

**Pull request merged**